Bottom: 09ed451c6b5c30d2dbe0bbedc3379ef8034dc899
Top:    ee02af812700683098831857a8b518d7b8109c39
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-04-16 21:42:06 -0700

rsocket: Check max inline data after creating QP

The ipath provider will ignore the max_inline_size
specified as input into ibv_create_qp and instead
return the size that it supports (which is 0) on
output.  Check that the max_inline_size returned
from ibv_create_qp is the size needed.

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/src/cma.c b/src/cma.c
index 0dc229e..4fa41ff 100644
--- a/src/cma.c
+++ b/src/cma.c
@@ -321,7 +321,7 @@ err:
 	return ret;
 }
 
-int ucma_init_all(void)
+static int ucma_init_all(void)
 {
 	int i, ret = 0;
 
diff --git a/src/cma.h b/src/cma.h
index a7bab0f..4c991b4 100644
--- a/src/cma.h
+++ b/src/cma.h
@@ -158,7 +158,7 @@ static inline int ERR(int err)
 	return -1;
 }
 
-int ucma_init(void);
+int ucma_init();
 extern int af_ib_support;
 
 #define RAI_ROUTEONLY		0x01000000
